metadata-ingestion/src/datahub/ingestion/source/sql/oracle.py

```diff
@@ -85,6 +86,16 @@ class OracleConfig(BasicSQLAlchemyConfig):
         description="The data dictionary views mode, to extract information about schema objects "
         "('ALL' and 'DBA' views are supported). (https://docs.oracle.com/cd/E11882_01/nav/catalog_views.htm)",
     )
+    # oracledb settings to enable thick mode and client library location
+    thick_mode: Optional[bool] = Field(
```

Comment:
may you rename this to `enable_thick_mode`? 
`enable_*` is a quite common pattern in the configs